### PR TITLE
fixed #25440

### LINKED ISSUE #25440: Typo
https://github.com/symfony/symfony/blob/efc19fc6d7df6979ac34d7fd864370d722b1cb2c/src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php#L76

I believe, it should refer to ``$this->options['scope']`` (currently it refers ``$this->options['scopen']`` (extra ``n``)).

### PR DESCRIPTION
**Pull request by ptomulik**
| Q             | A
| ------------- | ---
| Branch?       | master for features / 2.7 up to 4.0 for bug fixes 
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | yes/no
| Deprecations? | no 
| Tests pass?   | yes/no
| Fixed tickets | #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Fixes #25440

### CODE REVIEW

File: src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function execute()
                     $func = 'ldap_search';
                     break;
                 default:
-                    throw new LdapException(sprintf('Could not search in scope %s', $this->options['scopen']));
+                    throw new LdapException(sprintf('Could not search in scope %s', $this->options['scope']));
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php]: would you ind adding double quotes around the "%s" please?

Comment (ptomulik) [reply src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php]: Done.

**Review by nicolas-grekas — APPROVED** [review]

**Review by Simperfit — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ptomulik.

**Comment by Simperfit** [comment]
Congrats on the first contribution @ptomulik !

**Pull request merged**